.github/workflows/run-unit-tests.yml

```diff
@@ -75,47 +75,56 @@ jobs:
       - name: Install & Start Docker
         if: env.INSTALL_DOCKER == '1'
         run: |
-          # Uninstall colima to upgrade to the latest version
-          if brew list colima &>/dev/null; then
-              brew uninstall colima
-              # unlinking colima dependency: go
-              brew uninstall go@1.21
-          fi
-          rm -rf ~/.colima ~/.lima
-          brew install --HEAD colima
-          brew install docker
-
-          export DOCKER_CONTEXT="colima-$GITHUB_RUN_ID"
-          docker context create $DOCKER_CONTEXT --docker host=unix:///var/run/docker.sock
+          IP_BASE="192.168.172."
 
           start_colima() {
-            # Find a free port
-            RANDOM_PORT=$((RANDOM % 16384 + 49152))
+            # Change IP address between attempts
+            local IP_SUFFIX=$1
+            local IP_ADDRESS="${IP_BASE}${IP_SUFFIX}"
 
-            echo "Using random port: $RANDOM_PORT for SSH"
+            # Find a free port in the range 10000-20000
+            RANDOM_PORT=$((RANDOM % 10001 + 10000))
 
-            colima start --network-address --arch x86_64 --cpu=1 --memory=1 --ssh-port $RANDOM_PORT
-            docker context use $DOCKER_CONTEXT
+            # Create Lima instance with the specific IP address
+            limactl create colima-${IP_ADDRESS}
+
+            if ! colima start --network-address $IP_ADDRESS --arch x86_64 --cpu=1 --memory=1 --verbose --ssh-port $RANDOM_PORT; then
```

Comment:
`network-address` is a boolean arg
https://github.com/abiosoft/colima/blob/c8c4c5a69b4e422dab76ac0a0f81515094302c2e/cmd/start.go#L160

Reply:
Again, you have to read the full source:
https://github.com/abiosoft/colima/blob/c8c4c5a69b4e422dab76ac0a0f81515094302c2e/cmd/start.go#L159-L161
```py
	if util.MacOS() {
		startCmd.Flags().BoolVar(&startCmdArgs.Network.Address, "network-address", false, "assign reachable IP address to the VM")
	}
```

Reply:
This line sets up a boolean command-line flag named network-address. When this flag is set to true by the user, it indicates that a reachable IP address should be assigned to the VM. By default, this flag is false. isn't it?

Reply:
🤕 ohh, yes, I did just read the text, not the "BoolVar" and misunderstood it. 
There is not enough ☕☕☕around, apparently.
Thanks for correcting me there!